config/routes.rb

```diff
@@ -1,12 +1,14 @@
 Rails.application.routes.draw do
   root 'welcome#index'
+
   mount_devise_token_auth_for 'User', at: 'auth'
   namespace :api do
     namespace :v1 do
-      resources :users
-      resources :organizations
+      resources :users, :venues, :organizations
```

Comment:
Why did you do that?

Reply:
![Screenshot from 2019-08-27 12-31-31](https://user-images.githubusercontent.com/40833597/63759717-a6db9900-c8c6-11e9-8d58-2f608586c87d.png)
common practice